.github/ISSUE_TEMPLATE/01-bug.yml

```
@@ -0,0 +1,60 @@
+name: General Bug
+labels: ["status/needs-triage", "type/bug"]
+description: Report a general problem with Bottlerocket
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This bug report form is for **general issues** with Bottlerocket. 
+        
+        A few tips before submitting a new issue:
+        - Search for similar issues, including [closed issues](https://github.com/bottlerocket-os/bottlerocket/issues?q=is%3Aissue+is%3Aclosed),
+        - Include details about the environment you're running in,
+        - Include any error messages you received, with any required context.
+  - type: dropdown
+    id: variant-family
+    attributes:
+      label: Variant family
+      options:
+        - aws-k8s
+        - aws-k8s (nvidia)
+        - aws-ecs
+        - aws-ecs (nvidia)
+        - vmware-k8s
+        - metal-k8s
+        - Custom variant
```

We could omit the (nvidia) ones I think; it'll be clear from the bug report whether GPUs are involved:
```suggestion
      options:
        - aws-k8s
        - aws-ecs
        - vmware-k8s
        - metal-k8s
        - custom
```

---

[Will be obsolete by changes to other questions]